### ISSUE TITLE
Cannot type capital "C" because of shift+c hotkey

### ISSUE THREAD

**Issue by iavael** [akamud/vscode-caniuse#1]
Cannot type capital "C" letter in editor, because shift+c keybinding catches it.

**Comment by akamud** [comment]
Wow, that was lousy of me. Thanks for reminding me. 

I think I'll stick with `ctrl+shift+i` for can "I" use, since "C" and "U" are already taken.

**Comment by akamud** [comment]
I'll update this as soon as I get back home. In the meantime you can clear the shortcut like this:

```Javascript
{ "key": "shift+c",             "when": "editorTextFocus" }
```

Sorry for the trouble.

**Comment by akamud** [comment]
This is now fixed in [0.3.1](https://marketplace.visualstudio.com/items/akamud.vscode-caniuse), sorry for the inconvenience and thanks for reporting.
